### DISCUSSION TITLE
Putting cybersecurity advocacy into action with GitHub Advanced Security [SEC1808M]

### DISCUSSION THREAD

**Discussion by marietheresa** [community/community#74695] [Enterprise]
## Putting cybersecurity advocacy into action with GitHub Advanced Security [SEC1808M]

**Speaker:** @marietheresa (Marie Theresa Brosig)
**Social:** You can follow me on www.linkedin.com/in/marie-theresa-brosig

> Security and development teams are often at odds. But there is a way to (metaphorically) reach across the table and come together to achieve a common goal: to write, build, and run your company’s software more securely. In this session, I’ll be using the rollout of GitHub Advanced Security (GHAS) to illustrate how you can apply cybersecurity advocacy concepts to build new relationships with developers and strengthen existing ones, and how to engage and interact with developers in a more developer-friendly way.

Thank you so much for taking an interest and being here! I'd love to hear your thoughts, comments and questions on my session - and I'll do my best to answer. Thank you again. 

To watch the recording, please click [tbd]. 
To find all the strategies and steps I talk about in my talk, go to the repository here: https://github.com/marietheresa/improved-lamp

Hope to see you in the comments.
Marie

**Comment by jricciardi** [comment]
Welcome to the main stage, @marietheresa! 
![Screenshot 2023-11-08 at 3 46 40 PM](https://github.com/community/community/assets/31521880/1d8f2c9a-ec92-43df-a2dd-7c5cfcae2af5)

**Reply by wanghan5204** [reply]
怎么加入电报群

**Reply by ramonesmith777** [reply]
https://www.amazon.com/dp/B0CMXKZ3M8?ref_=pe_3052080_397514860

**Reply by ramonesmith777** [reply]
https://www.amazon.com/dp/B0CMK6GGQ7?ref_=pe_3052080_397514860

**Reply by ramonesmith777** [reply]
My name is Ramone Smith and I'm a Certified Digital Career Strategist/Technical Career Coach. I'd like to share job searching and career development resources with you. Feel free to reach out if you have any questions. Also, feel free to reach out and book a career coaching session.

https://www.youtube.com/watch?v=KZPrvbNosKg

https://www.amazon.com/dp/B0CLK2TLFJ?ref_=pe_3052080_397514860

https://www.amazon.com/dp/B0CMK6GGQ7?ref_=pe_3052080_397514860

https://www.amazon.com/dp/B0CGKV5RQC?ref_=pe_3052080_397514860

https://www.youtube.com/watch?v=XLHx5aQ9xKY

https://www.youtube.com/watch?v=hkOJpYbeqQo

https://www.technicalcareerstrategistcoachingservices.com/product-page/career-development-curriculum-for-technical-coding-bootcamp-students

https://www.amazon.com/dp/B0CGKV5RQC?ref_=pe_3052080_397514860

https://www.amazon.com/dp/B0CLK2TLFJ?ref_=pe_3052080_397514860

https://www.youtube.com/watch?v=H_HM-ERtIn0

https://www.youtube.com/watch?v=I2FzRdnsTh0

https://www.youtube.com/watch?v=DdZZtgRal1A

https://ramonesmith15.wixsite.com/technical-career-str/product-page/career-development-curriculum-for-technical-coding-bootcamp-students

**Reply by ramonesmith777** [reply]
I hope you are all well. I'd like to share an additional job search resource with you all. It's called Pathrise. Below are two links that explain how it works along with the link for enrollment. Reach out if you have any questions. Continued good luck with your job search.

https://www.pathrise.com/apply?utm_source=ramone&utm_medium=affiliate&utm_campaign=apply_linkedin

https://www.youtube.com/watch?v=j6wd4431GIE

https://www.youtube.com/watch?v=rT0AekN6nag

**Comment by jricciardi** [comment]
A framework for How We Change - based on the [6-step transtheoretical model of change](https://sphweb.bumc.bu.edu/otlt/mph-modules/sb/behavioralchangetheories/behavioralchangetheories6.html).

<img width="1354" alt="Screenshot 2023-11-08 at 3 54 03 PM" src="https://github.com/community/community/assets/31521880/136e1078-7f85-4b7b-a6c9-44e72877d061">

**Comment by ghostinhershell** [comment]
How do you create awareness of, and demonstrate that a problem is a problem? Check out the issues that @marietheresa created to showcase the Pre-Contemplation flow:
![image](https://github.com/community/community/assets/14021469/4c771a52-b255-44bc-8699-f8fa12460033)

**Comment by jricciardi** [comment]
"The more you work through the change beforehand and experience the change yourself, the better rolling out the change will go"

![Screenshot 2023-11-08 at 4 11 22 PM](https://github.com/community/community/assets/31521880/9693f41a-a2c9-487a-bf21-38a98e9a86cc)

**Comment by jricciardi** [comment]
The essentials? Host workshops showing how to do things, step-by-step, allowing for Q&A. 'Expert'-level workshops are fun for the power users and tinkerers among us :) "Build a community around curiosity and customization"

And don't forget to also publicly celebrate accomplishments, like milestones and goals! "Highlight the hard work ... maybe have some cake"

![Screenshot 2023-11-08 at 4 18 11 PM](https://github.com/community/community/assets/31521880/c299d896-eb75-4c7e-9673-e08e35b5e7e4)

**Comment by jricciardi** [comment]
"If a tool is mandatory, I don't want you to hate using it" is a great quote 🙌

**Comment by flaxel** [comment]
Really nice presentation and insights about change management and the GitHub Advanced Security. 🙌 Would be really cool if you could make the repository public.

**Reply by jricciardi** [reply]
@flaxel it looks like the repo is now public 👍

**Comment by allergic-reaction** [comment]
Thank you!
